### ISSUE TITLE
Ubuntu WSL 16.04 Error Instalation

### ISSUE THREAD

**Issue by sgnd** [asdf-community/asdf-haskell#6]
gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now
install: cannot stat '/tmp/tmp.TUC2Ol5SoE/*/stack': No such file or directory
Installing Haskell 8.4.2 (ghc-8.4.2)
STACK_ROOT=/home/suganda/.asdf/installs/haskell/8.4.2/stack
/home/suganda/.asdf/installs/haskell/8.4.2/bin/stack: line 2: /home/suganda/.asdf/plugins/haskell/stack/stack: No such file or directory

**Comment by green4ao** [comment]
Same here with Antergos Gnome

**Comment by HuShifang** [comment]
Same here with Kubuntu 18.04 (attempted installing both 8.4.2 and 8.2.2, with the same results)

**Comment by vyorkin** [comment]
Same thing, 4.16.13-1-ARCH

**Comment by aditya7iyengar** [comment]
Same error on 4.17.2.1-ARCH

**Comment by waiting-for-dev** [comment]
Same with debian...

**Comment by mitizhi** [comment]
I traced this one, and found that there is an attempt to download tar ball from URL https://www.stackage.org/stack/linux-x86_64-static which gives 404. If you go one step up to https://www.stackage.org/stack with web browser, you see that there is a link from there to the failing URL. So stackage.org has a bug there too.

I tried changing the download version from "linux-x86_64-static" to "linux-x86_64" and things go past the failing point after that.
